doc/how_to/authentication/configuration.md

```diff
@@ -37,7 +37,7 @@ pn.serve(app, oauth_provider=...)
 The login and logout endpoints are configurable:
 
 ```
-panel serve oauth_example.py --login-endpoint="/signin" --logout-endpoint="/signoff"
+panel serve oauth_example.py --login-endpoint="/signin" --logout-endpoint="/signoff" ...
```

Comment:
I would have correct this documentation instead. Why have to add the `/`? Why not just provide the name of the endpoint?

Reply:
You don't have to I guess, but I think it's more explicit.

Reply:
Much like the root-path argument in FastAPI: https://fastapi.tiangolo.com/advanced/behind-a-proxy/#providing-the-root_path